lib/zendesk_apps_support/assets/src.js.erb

```diff
@@ -21,6 +21,7 @@ var app = ZendeskApps.defineApp(<%= iframe_only ? 'null' : 'source' %>)
     appVersion: <%= version.to_json %>,
     locationIcons: <%= location_icons.to_json %>,
     assetUrlPrefix: <%= asset_url_prefix.to_json %>,
+    logoAssetHash: <%= logo_asset_hash.to_json %>,
```

Comment:
@mtsolakiszen do we have a plan to regenerate these assets? src.js is normally only updated when the app is updated.

Reply:
@danielbreves The implementation has a fallback to use the current logic for retrieving the logo if the `logo_assets_hash` isn't defined therefore new apps will use the new method and all existing apps should still work